firefox-ios/firefox-ios-tests/Tests/ClientTests/SearchViewControllerTests.swift

```diff
@@ -87,7 +87,7 @@ class SearchViewControllerTest: XCTestCase {
 
     func testFirefoxSuggestionReturnsNoSuggestions() async throws {
         FxNimbus.shared.features.firefoxSuggestFeature.with(initializer: { _, _ in
-          FirefoxSuggestFeature(availableSuggestionsTypes: [.amp: false, .wikipedia: false])
+            FirefoxSuggestFeature(availableSuggestionsTypes: [.amp: false, .ampMobile: false, .wikipedia: false])
```

Comment:
Not sure if this extra space is correct indentation or not? Let's double check. lol. Also, this line looks long enough that Swiftlint might scream. If you've ran Swiftlint and it's hasn't that's fine. If not, either let's do that, or just break up this array onto new-lines.

Reply:
sooo yes, i found that weird looking too, but that seems like what xcode wants when i try to break the line there? swift lint locally wasn't screaming at me, but i agree the line is kinda long